x-pack/plugins/ml/public/application/routing/routes/datavisualizer/index_based.tsx

```diff
@@ -27,6 +27,9 @@ export const indexBasedRouteFactory = (
 ): MlRoute => ({
   id: 'data_view_datavisualizer',
   path: '/jobs/new_job/datavisualizer',
+  title: i18n.translate('xpack.ml.dataVisualizer.dataView.docTitle', {
+    defaultMessage: 'Date View Data Visualizer',
```

Comment:
typo `Date`

Reply:
fixed in https://github.com/elastic/kibana/pull/124125/commits/bc9b27bb7e9130483af5da1ebf433bc3bc4a0935